src/chrome/content/rules/novafilm.tv.xml

```diff
@@ -0,0 +1,37 @@
+<!--
+bb.novafilm.tv ⁵
+www.bb.novafilm.tv ⁵
+blog.novafilm.tv ⁵
+oc.s02e13.rus.eng.dvdrip.novafilm.tv ⁵
+feedback.novafilm.tv ¹
+malcolm.novafilm.tv ²
+morena.novafilm.tv ⁵
+ns1.novafilm.tv ⁵
+www.ns1.novafilm.tv ⁵
+ns2.novafilm.tv ⁵
+castle.s02e05.hdtv.rus.novafilm.tv ⁵
+castle.s02e07.hdtv.rus.novafilm.tv ⁵
+castle.s01e11.hdtvrip.rus.novafilm.tv ⁵
+supernatural.s05e06.hdtvrip.rus.novafilm.tv ⁵
+supernatural.s05e07.hdtvrip.rus.novafilm.tv ⁵
+supernatural.s05e08.hdtvrip.rus.novafilm.tv ⁵
+supernatural.s05e09.hdtvrip.rus.novafilm.tv ⁵
+supernatural.s05e10.hdtvrip.rus.novafilm.tv ⁵
+supernatural.s05e11.hdtvrip.rus.novafilm.tv ⁵
+shop.novafilm.tv ¹
+smsvote.novafilm.tv ⁵
+static.novafilm.tv ⁵
+video.novafilm.tv ⁵
+wash.novafilm.tv ⁵
```

Comment:
Same practice: I would prefer a comment about this behavior. The target list can be reduced to those hosts that are really in use. A list of hosts makes no sense here.